lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -458,7 +458,7 @@ def check_scope
           end
 
           def reload_position
-            self.reload
+            self.reload unless new_record?
```

Comment:
I'd like to remove it but I remember doing something like this before and then it got put back because it broke the way STI worked. But I think it was more to do with returning the class name for the AAL model.